client/src/components/pages/HomePage/SearchResult.tsx

```diff
@@ -100,9 +101,7 @@ const SearchResult = ({ article, position, queryId, queryTokens }: SearchResultP
   const [collapsed, setCollapsed] = useState<boolean>(true);
 
   // Separate abstract from other paragraphs if it was highlighted
-  const originalAbstract = article.highlighted_abstract
-    ? article.paragraphs[0]
-    : article.abstract || '';
+  const originalAbstract = article.abstract_html || '';
```

Comment:
rename this class to Cord19SearchResult and revert all the changes?

Reply:
i dont think you need to touch this since the user should just choose which type of search result to render